### PR TITLE
Add test to verify country boundaries

### LINKED ISSUE #18: add tests to ensure polygons satisfy GeoInterface

### PR DESCRIPTION
**Pull request by zerefwayne**

### CODE REVIEW

File: test/GADM.jl [hunk 1]
```diff
@@ -93,3 +93,21 @@ end
     feature_layer = GADM.getlevel(dataset, 0)
     @test typeof(feature_layer) === ArchGDAL.IFeatureLayer
 end
+
+@testset "correct geometry" begin
+
```
Comment (juliohm) [inline test/GADM.jl]: Do we really need the extra blank lines that you insist on adding? Remember to be picky with these things.

File: test/GADM.jl [hunk 2]
```diff
@@ -93,3 +93,21 @@ end
...
+    bounds_arr = [bounds.MinX, bounds.MaxX, bounds.MinY, bounds.MaxY]
+
+    # Vatican City bounding boxes lat min 41.9002044 lat max 41.9073912 lon min 12.4457442 lon max 12.4583653
+    bounds_actual = [12.4457442, 12.4583653, 41.9002044, 41.9073912]
+
+    for i = 1:4
+        diff = abs(bounds_arr[i] - bounds_actual[i])
+        @test diff < 0.01
+    end
+
```
Comment (juliohm) [inline test/GADM.jl]: Another extra line.

File: test/GADM.jl [hunk 3]
```diff
@@ -93,3 +93,21 @@ end
...
+    bounds_arr = [bounds.MinX, bounds.MaxX, bounds.MinY, bounds.MaxY]
+
+    # Vatican City bounding boxes lat min 41.9002044 lat max 41.9073912 lon min 12.4457442 lon max 12.4583653
+    bounds_actual = [12.4457442, 12.4583653, 41.9002044, 41.9073912]
+
+    for i = 1:4
+        diff = abs(bounds_arr[i] - bounds_actual[i])
+        @test diff < 0.01
+    end
```
Comment (juliohm) [inline test/GADM.jl:110]: Nice test :ok_hand:

File: Project.toml [hunk 4]
```diff
@@ -6,6 +6,7 @@ version = "0.1.0"
 [deps]
 ArchGDAL = "c9ce4bd3-c3d5-55b8-8973-c0e20141b8c3"
 DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
+GeoInterface = "cf35fbd7-0cd7-5166-be24-54bfbe79505f"
```
Comment (juliohm) [inline Project.toml]: For now let's make this a test dependency only. You can simply move this line to an `[extras]` section, and then list GeoInterface in the `[targets]` section. See this example: https://github.com/JuliaEarth/Variography.jl/blob/f7fd09aa0a705701bf6eb93c52294cc0a13a99ad/Project.toml#L32-L48

Comment (juliohm) [reply Project.toml]: You will then need to load GeoInterface explicitly in runtests.jl

**Review by juliohm — CHANGES_REQUESTED** [review]
Round of reviews attached :+1:

File: Project.toml (hunk 4 above)
Comment (zerefwayne) [reply Project.toml]: I forgot to remove this dependency. We don't need it anymore.
